core/src/main/java/kafka/log/remote/RemoteStorageMetrics.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package kafka.log.remote;
+
+import com.yammer.metrics.core.MetricName;
+import org.apache.kafka.server.metrics.KafkaYammerMetrics;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class RemoteStorageMetrics {
```

Comment:
I was hoping that we could add all metric related to Remote Storage here (and validate that they aren't initialized). Metrics which are missing are the ones such as REMOTE_LOG_MANAGER_TASKS_AVG_IDLE_PERCENT and others which are only applicable if Remote Storage is turned on at system level.

Reply:
Added `RemoteLogManagerTasksAvgIdlePercent`, `RemoteLogReaderTaskQueueSize`, and `RemoteLogReaderAvgIdlePercent`. Thanks.